.project

```diff
@@ -35,6 +35,11 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
+			<name>org.springframework.ide.eclipse.boot.validation.springbootbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
```

Comment:
Is it necessary to modify this root-level file?

Reply:
No! It was pushed by error, sorry.

Reply:
Now I removed it!